scripts/installMacDeps.sh

```diff
@@ -84,8 +84,7 @@ install_homebrew(){
   if [ "$CACHE" == "true" ]; then
     install_homebrew_from_cache
   fi
-  which -s brew
-  if [[ $? != 0 ]] ; then
+  if ! which -s brew; then
```

Comment:
can you provide more info about the issue you had please?

Reply:
in a clean install, sierra 10.13.4, when clicking enter on installing Homebrew nothing happens and exit the script...
It's quite strange but this fixed for me.